.travis.yml

```diff
@@ -7,6 +7,7 @@ branches:
   - master
   - develop
   - feature/simpleFacets
+  - search-enhancement
```

Comment:
Do you need to build in travis? The PR will run a build prior to merging so not sure this is needed - it will need to be removed from `develop` once merged too.

Reply:
wanted to deploy it to test, will remove this.

Reply:
Usually we wait to deploy it until its merged, but fair enough.

Reply:
also wanted to make sure no broken test cases.

Reply:
Can run the tests in IDE or command line - and it runs on travis when you create the PR.

![image](https://user-images.githubusercontent.com/532845/88513845-692e4580-d02c-11ea-9966-35e37b098573.png)